includes/class-wc-payments.php

```diff
@@ -231,7 +231,9 @@ public static function init() {
 
 		add_filter( 'woocommerce_payment_gateways', [ __CLASS__, 'register_gateway' ] );
 		add_filter( 'option_woocommerce_gateway_order', [ __CLASS__, 'set_gateway_top_of_list' ], 2 );
+		add_filter( 'option_woocommerce_gateway_order', [ __CLASS__, 'replace_wcpay_gateway_with_payment_methods' ], 3 );
```

Comment:
Are these changes accurate? It seems like they might be coming from a merge conflict 👀

Reply:
Let me check that!. Actually, 50% of the work in this PR was merging it into develop, so I wouldn't be surprised if I messed something during the merge.